pkg/docker/config/config.go

```diff
@@ -719,3 +742,14 @@ func normalizeRegistry(registry string) string {
 	}
 	return normalized
 }
+
+// isNamespaced returns true if the provided registry contains one or more
+// subpaths.
+func isNamespaced(registry string) bool {
+	stripped := stripScheme(registry)
+
+	// don't take trailling slashes into account
+	stripped = strings.TrimSuffix(stripped, "/")
+
+	return len(strings.Split(stripped, "/")) > 1
```

Comment:
This is just `containsRune(stripped, '/')`, isn’t it?

Reply:
Yep, fixed as suggested :+1: